build-packages.sh

```diff
@@ -110,8 +110,8 @@ export DEBEMAIL="Confluent Packaging <packages@confluent.io>"
 xmlstarlet ed --inplace -P --update "/_:project/_:parent/_:version" --value "${UPSTREAM_VERSION}" pom.xml
 
 # Maven provides some helpfull commands for setting versions/properties
-mvn --batch-mode versions:set          -DgenerateBackupPoms=false "-DnewVersion=${FULL_VERSION}"
-mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${FULL_VERSION}" -Dproperty=io.confluent.ksql.version 
+mvn --batch-mode versions:set          -DgenerateBackupPoms=false "-DnewVersion=${VERSION}"
```

Comment:
This I'm not sure about - I don't know how these maven commands are used

Reply:
This change is correct. We want the maven artifacts to be versioned as just `0.17.0`, not `0.17.0-1`.